mysql/config.go

```diff
@@ -47,6 +47,11 @@ type Config struct {
 	// Optional. Default is 10 * time.Second
 	GCInterval time.Duration
 
+	// CheckSchema checks that the "v" column data type is correct. See https://github.com/gofiber/fiber/issues/1258
+	//
+	// Optional. Default is true
+	CheckSchema bool
```

Comment:
![image](https://user-images.githubusercontent.com/7063188/116194578-4f439f80-a731-11eb-9451-a64fd52ed1d4.png)
when i add a config, which most do because of their password, the checkSchema switch is transmitted with false

few will consider it, here we need to find another solution